hw/ip/prim/dv/prim_ram_scr/cpp/scramble_model.h

```diff
@@ -0,0 +1,69 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_HW_IP_PRIM_DV_PRIM_RAM_SCR_CPP_SCRAMBLE_MODEL_H_
+#define OPENTITAN_HW_IP_PRIM_DV_PRIM_RAM_SCR_CPP_SCRAMBLE_MODEL_H_
+
+#include <stdint.h>
+#include <vector>
+
+const uint32_t kPrinceWidth = 64;
+const uint32_t kPrinceWidthByte = kPrinceWidth / 8;
```

Comment:
Do these need to be exposed in the header?

Reply:
They're used in `scrambled_ecc32_mem_area.cc` I thought it was better to have a single place where we define the width of PRINCE.

Reply:
Oh, ok. Make sense.